avocado/core/job.py

```diff
@@ -328,15 +328,21 @@ def _get_avocado_git_version():
             git = path.find_command('git')
         except path.CmdNotFoundError:
             return
-        git_root = process.run('%s rev-parse --show-toplevel' % git,
-                               ignore_status=True, verbose=False)
-        if git_root.exit_status == 0 and os.path.exists(os.path.join(
-                git_root.stdout.strip(), 'python-avocado.spec')):
-            cmd = "%s show --summary --pretty='%%H'" % git
-            res = process.run(cmd, ignore_status=True, verbose=False)
-            if res.exit_status == 0:
-                top_commit = res.stdout.splitlines()[0][:8]
-                return " (GIT commit %s)" % top_commit
+        # We need to get git root as `py2to3` creates this file in BUILD
+        olddir = os.getcwd()
+        try:
+            os.chdir(os.path.dirname(__file__))
```

Comment:
OK, I tested this on Python 3, and it looks like `git rev-parse` has no issues at all with being called from `build/*`, it will still find the right top most directory that is tracked by GIT.